.github/workflows/module_switcher

```diff
@@ -0,0 +1,116 @@
+switch_for_module ()
```

Comment:
@skosukhin I do not understand what this module is doing. I understood that this is taken from another CI instance running at CSCS? Could you add a line or three of commentary to explain this?

Reply:
I have added a comment in the file about what the shell function does.